examples_tests/base.go

```diff
@@ -86,7 +88,7 @@ func randNameGen(format string) func() string {
 }
 
 func newClient(token string) (*aiven.Client, error) {
```

Comment:
lgtm, but please, comment all functions / type definitions (e.g. structs) / top level constants or variables, if any

Reply:
Please configure your IDE or tell me what functions needs docs and why. Comments without sense are waste of time.